CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # CKEditor 4 WYSIWYG Editor React Integration Changelog
 
+## ckeditor4-react 2.0.0-rc.1
+
+Other Changes:
+
+* Added CHANGELOG entries for RC versions.
+* Improved project README.
+
+## ckeditor4-react 2.0.0-rc.0
+
+BREAKING CHANGES:
+
+* [#124](https://github.com/ckeditor/ckeditor4-react/issues/124): Introduced support for React hooks and rewrote the component to use hooks internally.
+
+New Features:
+
+* [#159](https://github.com/ckeditor/ckeditor4-react/issues/159): Introduced support for React 17+ versions.
+* [#82](https://github.com/ckeditor/ckeditor4-react/issues/82): Introduced TypeScript support.
+* [#180](https://github.com/ckeditor/ckeditor4-react/issues/180): Introduced support for consumption of a "non-webpacked" package version.
```

Comment:
Sounds a little bit odd… Maybe "not bundled" or just "source of the package"?

Reply:
Additionally it doesn't seem that the package publishes `src` → https://npm-explorer.tk/?p=ckeditor4-react@2.0.0-rc.0

Reply:
We talked F2F with @Comandeer and concluded that source can be consumed thanks to multiple package formats, and I also adjusted entry stating just that - "Introduced support for consumption of a not bundled package version by providing package in ESM, CJS and UMD formats."